plugins/generic/externalFeed/classes/ExternalFeedDAO.inc.php

```diff
@@ -92,7 +103,7 @@ function &_returnExternalFeedFromRow($row) {
 	 * @param $externalFeed ExternalFeed
 	 * @return int 
 	 */
-	function insertExternalFeed(&$externalFeed) {
+	public function insertExternalFeed(&$externalFeed) {
```

Comment:
Our CRUD function name conventions have changed. Recent code uses `insertObject`, `getById`, `updateObject`, and `deleteObject`. This reference is also unnecessary.